src/components/ToolbarButton.tsx

```diff
@@ -54,11 +54,22 @@ const ToolbarButtonComponent: FC<DraggableToolbarButtonProps> = ({
     throw new Error('The svg property is required to render a shortcut in the Toolbar. ' + shortcutId)
   }
 
-  const commandState = commandStateStore.useSelector(state => state[shortcutId])
   const isDraggingAny = useSelector(state => !!state.dragShortcut)
   const dragShortcutZone = useSelector(state => state.dragShortcutZone)
-  const isButtonActive =
-    useSelector(state => (customize ? selected : !isActive || isActive(() => state))) || (!customize && commandState)
+
+  const commandState = commandStateStore.useSelector(state => state)
+  const isButtonActive = useSelector(state => {
```

Comment:
`isButtonActive` will only update when the state changes, not when `commandState` changes from what I can tell. `useSelector` is only triggered when the Redux state changes.